crates/sargon/src/gateway_api/methods/transaction_methods.rs

```diff
@@ -1,5 +1,6 @@
 use crate::prelude::*;
 use radix_engine_toolkit_common::receipt::SerializableToolkitTransactionReceipt;
+use radix_transactions::model::IntentHash;
```

Comment:
@matiasbzurovski delete this and use `ScryptoIntentHash` instead I think

Reply:
actually this import isn't necessary, since we need to use our internal `SubintentHash` (and not the one from `radix_transactions`).

I believe RustRover auto imported it when copy-pasting some code, will keep an eye on this